docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -305,6 +305,26 @@ document
 
 </PlatformSection>
 
+### Loading Strategies
+
+The `feedbackIntegration` is special among our integrations because it is a user-facing integration. Therefore, we have two strategies available to help you control bundle size. What strategies are available depend on how you have installed the SDK into your application.
+
+<Note>
+In every case, we recommend using `feedbackIntegration` in your `Sentry.init` call.
+</Note>
+
+1. `feedbackSyncIntegration`
+
+    This integration includes all the code needed to render the "Send Feedback" button, as well as the form that is triggered when the button is clicked. Choosing this loading strategy means accepting the largest bundle size in your application. The benefit of this is that the feedback widget is guaranteed to work when the user interacts with it.
+
+    This loading strategy is not available if you have installed the SDK via the CDN.
+
+2. `feedbackAsyncIntegration`
+
+    This integration includes the minimum amount of code needed to show the "Send Feedback" button on the screen when the page is loaded. When the user clicks on that button the integration will asyncronously load inrernal integrations that show the form, and let the user type out their feedback message. This has the benefit of being the lowest bundle size. The drawback is that asyncronous loading can fail when the user interacts with the button.
```

Comment:
```suggestion
    This integration includes the minimum amount of code needed to show the "Send Feedback" button on the screen when the page is loaded. When the user clicks on that button the integration will asynchronously load internal integrations that show the form and let the user type out their feedback message. This has the benefit of being the lowest bundle size. The drawback is that asynchronous loading can fail when the user interacts with the button.
```

Reply:
Maybe mention ad blocker as an example?
Should we describe how/where the async code is loaded from?

Reply:
i put some notes in there about loading from our CDN and ad-blockers. take another look at the whole paragraph and lmk